templates/web/base/admin/reports/_edit_waste.html

```diff
@@ -45,12 +45,15 @@
 
 <li><label class="inline-text" for="external_id">[% loc('External ID') %]:</label>
     <input type="text" class="form-control" name="external_id" id="external_id" value="[% problem.external_id | html %]">
-
+[% SET readonly = '' %]
+[% IF !c.cobrand.moniker == 'peterborough' %]
```

Comment:
This confused my brain when I first read it! Might be clearer written as `c.cobrand.moniker != 'peterborough'`, perhaps?

Or maybe clearer still would be `[% SET readonly = 'readonly' %]` initially (the default) and then set readonly to blank only if the cobrand is Peterborough (the exception)?